rj_gameplay/classifier/fumble_classifier.py

```diff
@@ -0,0 +1,123 @@
+import stp.rc as rc
+import rj_gameplay.gameplay_node as gameplay_node
+import rclpy
+
+
+def fumble_classifier(world_state: rc.WorldState, possess) -> bool:
+	if possess is None:
+		return True
+	else: 
+		return False
+
+"""
+get_distance returns the distance between the ball and the kicker
+num: kicker's id
+"""
+def get_distance(world_state: rc.WorldState, kicker) -> float:
+	if world_state is not None:
+		robot = world_state.our_robots[kicker]
+		pose_x = robot.pose[0]
+		pose_y = robot.pose[1]
+		ball_pos_x = world_state.ball.pos[0]
+		ball_pos_y = world_state.ball.pos[1]
+		distance = ((ball_pos_x-pose_x)**2 + (ball_pos_y-pose_y)**2)**0.5
+		return distance
+
+"""
+get_velocity_angle returns the ratio of y component of the velocity vector to x component
+"""
+
+def get_velocity_angle(world_state: rc.WorldState) -> float:
+	if world_state is not None:
+		ball_vel = world_state.ball.vel
+		vel = (ball_vel[0]**2 + ball_vel[1]**2) ** 0.5
+		if ball_vel[0] == 0:
+			angle = ball_vel[1] / (ball_vel[0]+0.00001)
+		else:
+			angle = ball_vel[1] / ball_vel[0]
+		return angle
+
+""" ball_possession returns the id of the robot who has the ball"""
+
+def ball_possession(world_state: rc.WorldState):
+	for robot in world_state.our_robots:
+		if robot.has_ball:
+			num = robot.id
+			return num
+		else:
+			return None
+
+"""
+fumbling: boolean whether it is fumbled or not
+"""
+def pass_or_fumble(world_state: rc.WorldState, kicker, old_angle, new_angle):
```

Comment:
A bit of a small thing, but don't forget to include the types for the parameters as well as the return types for functions.